### PR TITLE
Use OSS seek-io

### PR DESCRIPTION
**Pull request by ellisjoe**

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -52,6 +52,10 @@ subprojects {
 
     repositories {
         jcenter()
+        maven {
+            // TODO(jellis): figure out why seek-io and hadoop-crypto aren't mirrored to jcenter
```
Comment (schlosna) [inline build.gradle]: Seems like seek-io is still private, that could be why.

**Comment by uschi2000** [comment]
Don't think that's the reason. Did you check the 'publish to jcenter'
button in bjntray for this project?

**Comment by ellisjoe** [comment]
found the add to jcenter button. said it sent a message to jcenter...

**Comment by ellisjoe** [comment]
@uschi2000 should be good now

File: versions.props [hunk 2]
```diff
@@ -1,5 +1,6 @@
 com.fasterxml.jackson.*:jackson-* = 2.6.1
 com.google.guava:guava = 18.0
+com.palantir.seek-io:seek-io = 0.0.2
```
Comment (uschi2000) [inline versions.props]: nit: would start with 0.1.0, but doesn't matter too much.

Comment (ellisjoe) [reply versions.props]: done

**Review by markelliot — APPROVED** [review]

**Pull request merged**